### PR TITLE
Change default port to 11633

### PR DESCRIPTION
**Pull request by schoren**
This PR changes the default port to 11633

## Checklist

- [x] tested locally
- [ ] added new dependencies
- [ ] updated the docs
- [ ] added a test

### CODE REVIEW

File: .github/workflows/pull-request.yaml [hunk 1]
```diff
@@ -287,25 +282,20 @@ jobs:
         run: |
           kubectl port-forward \
             --namespace tracetest-integration \
-            svc/tracetest-integration 8080 &
+            svc/tracetest-integration 11633:8080 &
```
Comment (mathnogueira) [inline .github/workflows/pull-request.yaml:285]: Won't this cause a problem with the CLI? You are configuring the CLI to send commands to `localhost:11633`, but here tracetest will be available on port `8080`.

Comment (mathnogueira) [inline .github/workflows/pull-request.yaml:285]: You will have to update this right after the PR is merged, right?

Comment (schoren) [reply .github/workflows/pull-request.yaml:285]: Yes. exactly. `integration` is still running on `8080`, so I need to talk to it, but leave it as ready as possible for later.
So this part is temporary until this PR is merged, and then I have to make a new PR to correctly point to integrations's `11633` port

**Review by mathnogueira — APPROVED** [review]
LGTM

**Review by jfermi — APPROVED** [review]

**Pull request merged**